scripts/mutation-probe.py

```
@@ -2011,6 +2011,26 @@ def behavioral_verdict_title_diagnostic(
+        "packages/store-postgres/src/fragments.ts",
+        "export const jsonbInputValid = (value: string): string => `pg_input_is_valid(${value}, 'jsonb')`",
+        "export const jsonbInputValid = (_value: string): string => `TRUE` // MUTATION",
+        "PostgreSQL casts syntactically valid but jsonb-inadmissible durable JSON and aborts a bounded claim",
+    ),
+    (
+        "postgres-retry-factor-type-guard",
+        "packages/store-postgres/src/fragments.ts",
+        "  const factorAdmissible = `(CASE\n"
+        "      WHEN jsonb_typeof(${retry}::jsonb -> 'factor') <> 'number' THEN FALSE\n"
+        "      ELSE ${factor} BETWEEN 0 AND 1.7976931348623157e308\n"
+        "    END)`",
```

Ruff (0.16.2)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)